<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="db" uuid="382823f9-b78d-4ae4-9375-0336f9d9299c">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:$PROJECT_DIR$/PyTraker/tracker/db.sqlite3</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="Django default" uuid="ef57ab83-4976-4264-b872-43e7a19b4b43">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/PyTraker/tracker/tracker/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:$PROJECT_DIR$/PyTraker/tracker/db.sqlite3</jdbc-url>
    </data-source>
  </component>
</project>